doc/anjuta.1
.TH "Anjuta" "1" "0.1.10-pre" "Rob Bradford" ""
.SH "NAME"
.LP 
anjuta \- GNOME Integrated Development Environment
.SH "SYNTAX"
.LP 
anjuta [\fIoption\fP]
.SH "DESCRIPTION"
.LP 
Anjuta is a GNOME IDE and can be used to create either GNOME/GTK+ applications with glade or glade\-gnome, or can be used for creating generic applications. It is primarily designed for use with C/C++ and features an easy to use debugger and compilation environment.
.SH "OPTIONS"
.LP 
\fB\-\-help, \-h\fR
Output help information and exit.
.TP 
\fB\-\-version\fR
Output version information and exit.
.TP 
Anjuta supports all the generic GNOME and GTK options, a full list is available on the Anjuta help screen.
.SH "AUTHORS"
.LP 
This man page was written by Rob Bradford <robster@debian.org>. Anjuta was developed by Kh. Naba Kumar Singh <kh_naba@users.sourceforge.net>
.SH "SEE ALSO"
.LP 
anjuta_launcher(1)


NEWS
Anjuta 3.1.92 "Hot summer in Nagano" (18 September 2011) -- Johannes Schmid

Bugs fixed:
656470 	Dangling symlinks created by make install
658229 	Inconsistent size of widgets in Properties Frames of Class Generator Dialog
658262 	Extra separator between double upper case characters
566700 	Default font for the Scintilla editor
654320 	Vala: New Class Wizard - created file not added to project
655295 	Add Target in Vala project does nothing.
658903 	Anjuta Plugin wizard fails
654463 	Disable "Install missing packages" button on OpenBSD

Thanks to:
Akom Chotiphantawanon, Alexander Shopov, Antonio Fernandes C. Neto,
Arnel A. Borja, Bruce Cowan, Chao-Hsiung Liao, Duarte Loreto, Javier Jardón, 
Jiro Matsuzawa, Joan Duran, Jorge González, Paul Seyfert, Sébastien Granjoux,
Yinghua Wang

Anjuta 3.1.91 "On the way to Tokio" (5 September 2011) -- Johannes Schmid

Updates:
 - More improvements in vala plugin (Abderrahim Kitouni)
 - Fix up problems with gtk+ 3.0 (Sébastien Granjoux)

Bugs fixed:
638525 	vala support crash on reading source file
657491 	Only parse the source files of the current target
657513 	anjuta fails to build in gnome 3.1
657589 	tests fail in tarball releases because tarballs are missing anjuta.lst
655756 	User should be notified if execution of a project fails
656123 	plural forms needed
657590 	Projects with libtool fails to regenerate
657022 	make clean removes libanjuta-interfaces.h which breaks make clean; make

Anjuta 3.1.90 "Whitsunday Islands" (29 August 2011) -- Johannes Schmid

Changes:
 - Some seldom used debugger menu items were removed (Sébastien Granjoux)
 - A lot of fixes to the vala support (Abderrahim Kitouni)
 - vala-0.14 (>= 0.13.3) is now required

Bugs fixed:
656584 	Error in DevHelp
642187 	C++ symbols are not shown in symbol browser
657023 	Build broken with vala-0.13.x
566704 	Better UI for environment variables
652047 	Support linking against libgda-5.0
656585 	<FileName>.ui and INSTALL not created in a new project
657134 	Missing file in POTFILES.in
653259 	Empty lines in Locals during debugging

Thanks to:
Abderrahim Kitouni, Aron Xu, Daniel Mustieles, Dominique Leuenberger,
Jorge González, Kjartan Maraas, Marek Černocký, Og B. Maciel, P. F. Chimento, 
Piotr Drąg, Sébastien Granjoux

Anjuta 3.1.5 "Sydney Edition" (13 August 2011) -- Johannes Schmid

Officially brought to you about 20 000 km away from desktop-summit!

Bugs fixed:
654881 	Segmentation fault during project creation if no packages were selected
651151 	vala plugin crashes when uncommenting source
654502 	Allow building with libvala-0.14 (0.13 development tree)
655743 	Usage of GtkHBoxClass and GtkHBox
655998 	Main toolbar looks out-of-place
655584 	Can't edit Integer Field

Thanks to: Abderrahim Kitouni, Akom Chotiphantawanon, Alban Browaeys,
Alexander Shopov, Daniel Mustieles, Dominique Leuenberger, Mario Blättermann, 
Piotr Drąg, Sébastien Granjoux, Yuri Myasoedov

Anjuta 3.1.4 "Leaving on a jet-plane" (25 July 2011) -- Johannes Schmid

Bugs fixed:
654502 	Allow building with libvala-0.14 (0.13 development tree)
646410 	Missing infrastructure for the new Mallard manual
654073 	Please apply the kfreebsd patch to anjuta-extras
654124 	Vala: add class wizard creates file twice.
654445 	Add 'BSD' as a valid license in the Class Generator
654099 	file manager plugin: fix leak for file
654100 	file manager plugin: fix leak in row callapsed
654559 	Weird pseudo-password dialog shown with half a commit message when I import a project
654092 	Crash when adding a project file
654095 	increase reference count of the selected file
654560 	Contents of Preferences tabs seem to be packed to fill all vertical space, with ugly consequences
631157 	unix is not defined on OpenBSD
654455 	Fix build on OpenBSD

Thanks to:
Abderrahim Kitouni, Alban Browaeys, Daniel Mustieles, Dominique Leuenberger,
Francisco Javier Fernández Serrador, Francisco Javier Serrador, 
Jasper Lievisse Adriaanse, Jorge González, Marek Černocký, ario Blättermann,
Piotr Drąg, Sébastien Granjoux, Tamara Atanasoska

Anjuta 3.1.3 "Is this it?" (4 July 2011) -- Johannes Schmid

New:
 - Various improvements in the project wizard
 - Git usuability improvements
 - Auto-completion for Gtk.Builder objects in python plugin
 - more mallard documentation

Bugs fixed:
649107 	Add an index of help topics to the Mallard manual
651212 	Auto-completion of Gtk.Builder objects
312667 	Why not allow people to compile a simple .cpp file not starting a project in a first place?
588952 	build-menu disabled if sources are in subfolder
590323 	Make the context menu "context"-aware
645516 	Enabled (translation comment)
646073 	Application Manager
646192 	New File (translation comment)
652773 	Anjuta (current git) fails to load abiword project
652778 	Opening an automake based project with empty configure.in file fails.
653364 	build failure on kfreebsd: Error on line 6 char 104: Invalid UTF-8 encoded text in
647403 	Selecting Configure External Packages and selecting no packages causes segfault.

Thanks to: Daniel Mustieles, Daniel Nylander, James Liggett, Johannes Schmid, 
Jorge González, Massimo Corà, Matej Urbančič, Philip Chimento, Sébastien Granjoux

Anjuta 3.1.2 "Berg 2011" (13 June 2011) -- Johannes Schmid

New:
- Autocompletion for builder objects in python (Jason Siefken)
- Improvements in autotools backend (Sebastien Granjoux)

Bugs fixed:
645199 	Signal dnd issues
475261 	Change install directory
563856 	generic Vala project type
651670 	Glade file not rendering
646530 	Anjuta crashes when closing tab other than current
588387 	symbol-db-view-locals: better search
651285 	Memory leak in symbol-db/readtags.c
569994 	support includes in makefiles
644047 	No Symbols shown in Global and Search Symbol pane
645659 	Basic aesthetic improvements to preferences dialog
645695 	install broken since deec63dfe4c2527d5adfed049eeac423307229d1
645965 	Theming tweaks
588757 	automake projects reject accented characters in paths
640470 	Port Scintilla to Gtk+3

www/htdocs/anjuta.php

<html>
<head>
	<title>Anjuta Integrated Development Environment</title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
	<meta content="programming, best, ide, free, user friendly, gnome, linux, anjuta, c, c++, php, versatile, gtk, website" name="keywords"/>
        <link href="/css/docs.css" type="text/css" rel="StyleSheet"/>
        <link href="/css/layout.css" rel="stylesheet" type="text/css" media="screen">
        <link href="/css/style.css" rel="stylesheet" type="text/css" media="all">
</head>

<body>
  <!-- site header -->
  <div id="page">
    <ul id="general">
      <li id="siteaction-gnome_home" class="home">
        <a href="http://anjuta.org/" title="Home">Home</a>
      </li>
      <li id="siteaction-gnome_news">
        <a href="http://sourceforge.net/news/?group_id=14222" title="News">News</a>
      </li>
      <li id="siteaction-gnome_projects">
        <a href="http://sourceforge.net/projects/anjuta" title="Project Site">Project Site</a>
      </li>
      <li id="siteaction-gnome_art">
        <a href="http://sourceforge.net/project/showfiles.php?group_id=14222" title="Releases">Releases</a>

      </li>
      <li id="siteaction-gnome_support">
        <a href="http://sourceforge.net/mail/?group_id=14222" title="Mailing lists">Mailing lists</a>
      </li>
      <li id="siteaction-gnome_development">
        <a href="http://sourceforge.net/forum/?group_id=14222" title="Forums">Forums</a>
      </li>
    </ul>

    <div id="header">
       <h1>Anjuta DevStudio: GNOME Integrated Development Environment</h1>
       <div id="tabs">
	<ul id="portal-globalnav">
		<li <?php if($_GET["page"] == "") echo("class=\"selected\"");?>><a href="/"><span>About</span></a></li>
		<li <?php if($_GET["page"] == "development") echo("class=\"selected\"");?>><a href="/development"><span>Development</span></a></li>
		<li <?php if($_GET["page"] == "downloads") echo("class=\"selected\"");?>><a href="/downloads"><span>Downloads</span></a></li>
		<li <?php if($_GET["page"] == "screenshots" || $_GET["page"] == "screen-shots" ) echo("class=\"selected\"");?>><a href="/screen-shots"><span>Screenshots</span></a></li>
		<li <?php if($_GET["page"] == "features") echo("class=\"selected\"");?>><a href="/features"><span>Features</span></a></li>
		<li <?php if($_GET["page"] == "credits") echo("class=\"selected\"");?>><a href="/credits"><span>Credits</span></a></li>
	</ul>
     </div> <!-- end of #tabs -->
    </div> <!-- end of #header -->
  </div> <!-- end of #page -->

<!-- end site header -->

<div id="sidebar">
	<h3>Project</h3>
	<ul>
		<li><a href="/changelog">ChangeLog</a></li>
		<li><a href="/roadmap">Roadmap</a></li>
		<li><a href="/hacking">Hacking</a></li>
		<li><a href="/tasks">Tasks & Bounties</a></li>
	</ul>
	<h3>Documentations</h3>
	<ul>
		<li><a href="http://library.gnome.org/devel/libanjuta/stable">Anjuta API docs</a></li>
		<li><a href="http://live.gnome.org/Anjuta">Anjuta wiki</a></li>
		<li><a href="http://library.gnome.org/devel/anjuta-faqs/stable">FAQ [English]</a></li>
		<!--<li><a href="/documentations/subpage/documents/ja/anjuta-faqs/index.html">FAQ [Japanese]</a></li>
		<li><a href="/documentations/subpage/documents/C/anjuta-tutorial/index.html">Tutorial [English]</a></li>
		<li><a href="/documentations/subpage/documents/de/anjuta-tutorial/index.html">Tutorial [German]</a></li>
		<li><a href="/documentations/subpage/documents/zh_CN/anjuta-tutorial/index.html">Tutorial [Chinese]</a></li> -->
		<li><a href="/documentations/subpage/documents/C/anjuta-build-tutorial/index.html">Build Tutorial [English]</a></li> 
		<li><a href="http://library.gnome.org/devel/anjuta-manual/stable">Manual [English]</a></li>
		<!--<li><a href="/documentations/subpage/documents/ja/anjuta-manual/index.html">Manual [Japanese]</a></li> -->
	</ul>
	<h3>Project Status</h3>
        <ul>
        <li><a href="http://bugzilla.gnome.org/browse.cgi?product=anjuta">Bugs</a></li>
        <li><a href="http://bugzilla.gnome.org/reports/patch-report.cgi?product=anjuta">Patches</a></li>
        <li><a href="http://bugzilla.gnome.org/simple-bug-guide.cgi?product=anjuta">Submit</a></li>
        </ul>
        <p>
	<a href="http://sourceforge.net/donate/index.php?group_id=14222"><img src="http://images.sourceforge.net/images/project-support.jpg" width="88" height="32" border="0" alt="Support This Project" /></a>
        </p>
        <p>
        <A href="http://sourceforge.net"><IMG
           src="http://sourceforge.net/sflogo.php?group_id=14222&amp;type=5"
           width="210" height="62" border="0" alt="SourceForge.net Logo"/></A>
        </p>
</div> <!-- sidebar -->

<div id="body">
<div id="content">
	<!-- Content starts here -->
	<?php
		$page = $_GET["page"];
		if ($page == "credits"){
			require("authors.php");
		} else
		if ($page == "development"){
			require("development.php");
		} else
		if ($page == "downloads"){
			require("downloads.php");
		} else
		if ($page == 'screenshots' || $page == 'screen-shots'){
			require("screenshots.php");
		} else
		if ($page == 'features'){
			require("features.php");
		} else
		if ($page == 'changelog'){
			require("changelog.php");
		} else
		if ($page == 'roadmap'){
			require("roadmap.php");
		} else
		if ($page == 'hacking'){
			require("hacking.php");
		} else
		if ($page == 'tasks'){
			require("tasks.php");
                } else
                if ($page == 'documentations'){
		  $subpage = $_GET["subpage"];
		  require("proxy.php");
		} else
		{
			require("home.php");
		}
	?>
</div> <!-- content -->
</div> <!-- body -->

</body>
</html>
